OrchardCore/Initialize-OrchardCore/Initialize-OrchardCore.psm1

```diff
@@ -1,10 +1,10 @@
 ﻿<#
 .Synopsis
-   Initializes an Orchard Core solution for a git repository. Deprecated, use Init-OrchardCoreSolution instead.
+   Initializes an Orchard Core solution for a git repository. Deprecated, use Initialize-OrchardCoreSolution instead.
 #>
 
 
-function Init-OrchardCore
+function Initialize-OrchardCore
```

Comment:
Keep this, since this is a proxy for an old name.

Reply:
Should I rename the file, or just add `[alias("Init-OrchardCore")]`?

Reply:
Alias is OK too.

Reply:
Yep, `Initialize` is the approved verb and `Init` is not recommended, but keeping it for backwards-compatibility is fine. Same for `Initialize-OrchardCoreSolution`.
https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands